### PR TITLE
Updated Schema with most recent version of Spreadsheet

### PR DESCRIPTION
**Pull request by krausf**
This PR adds the most recent changes from the MVP spreadsheet.

Added:

- Ontologies for certain properties. Defined in slot section
- Cleaning up duplicated entries

### CODE REVIEW

File: src/schema/ghga.yaml [hunk 1]
```diff
@@ -1521,54 +1542,94 @@ slots:
...
+      Method or device employed for collecting/isolating a sample.
+    related_mappings:
+      - NCIT:C25549
 
   storage:
-    description: Methods by which a sample is stored.
+    description: >-
+      Methods by which sample is stored 
+      (e.g. frozen in liquid nitrogen).
+    related_mappings:
```
Comment (deepakunni3) [inline src/schema/ghga.yaml:1581]: close mappings.

File: src/schema/ghga.yaml [hunk 2]
```diff
@@ -1521,54 +1542,94 @@ slots:
...
+    close_mappings:
+      - NCIT:C25717
 
   ethnicity:
-    description: Ethnicity of a person.
+    description: >-
+      A social group characterized by a distinctive social and cultural tradition 
+      that is maintained from generation to generation.
+    related_mappings:
+      - SIO:001014
```
Comment (deepakunni3) [inline src/schema/ghga.yaml:1602]: This can be exact mappings.

File: src/schema/ghga.yaml [hunk 3]
```diff
@@ -1613,31 +1680,44 @@ slots:
     description: The semantic type for the value of an attribute.
 
   affiliation:
-    description: Affiliations associated with an instance.
+    description: >-
+      Institutions associated with this study.
+    related_mappings:
```
Comment (deepakunni3) [inline src/schema/ghga.yaml:1685]: Close mappings.

File: src/schema/ghga.yaml [hunk 4]
```diff
@@ -1613,31 +1680,44 @@ slots:
...
   library type:
     description: >-
       Describe the level of omics analysis (eg: Metagenome, transcriptome, etc) 
 
   library selection:
     description: >-
-      Library Selection method (e.g. random, PCA, cDNA, etc )
+        Whether any method was used to select for or against, enrich, or screen the material being sequenced.
+        Library Selection method (e.g. random, PCA, cDNA, etc )
+    broad_mappings:
```
Comment (deepakunni3) [inline src/schema/ghga.yaml:1707]: Can be close mappings.

File: src/schema/ghga.yaml [hunk 5]
```diff
@@ -1657,23 +1739,34 @@ slots:
...
+      The strandedness of the library, whether reads come from both strands of the cDNA 
+      or only from the first (antisense) or the second (sense) strand.
+    broad_mappings:
+      - NCIT:C48673
   
   sequencing center: 
     description: >-
       Center where sample was sequenced.
+    close_mappings:
+      - NCIT:C172274
```
Comment (deepakunni3) [inline src/schema/ghga.yaml:1769]: `NCIT:C172274 Sequencing Platform Name` is conceptually different from `sequencing center`. The latter is referring to an insitution/site that hosts one (or more) sequencing instruments.

`GENEPIO:0000070 sequencing location` is a better mapping.

**Review by deepakunni3 — CHANGES_REQUESTED** [review]
Few recommendations added inline.

File: src/schema/ghga.yaml [hunk 6]
```diff
@@ -275,10 +281,31 @@ classes:
         description: >-
           Name for the experiment (eg: GHGAE_PBMC_RNAseq).
         required: true
+        broad_mappings:
+          - NCIT:C181729
```
[the remark this answers is not in the document]
Comment (krausf) [reply src/schema/ghga.yaml:285]: https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C181729

It is there but somehow not resolved...

File: src/schema/ghga.yaml [hunk 7]
```diff
@@ -602,28 +661,22 @@ classes:
...
         description: >-
-          Name of the Sample.
+          Name of the sample (eg:GHGAS_Blood_Sample1 or GHGAS_PBMC_RNAseq_S1).
         required: true
       type:
         description: >-
           Type of sample: 'Case' or 'Control'.
         range: case_control_enum
+        related_mappings:
+          - NCIT:C15197
```
[the remark this answers is not in the document]
Comment (krausf) [reply src/schema/ghga.yaml:671]: delete

**Review by deepakunni3 — APPROVED** [review]

**Pull request merged**